### ISSUE TITLE
Pybids doesn't detect the files in derivatives

### ISSUE THREAD

**Issue by Peyman-N** [openMetadataInitiative/bids2openminds#42]
The `pybids` seams ignores the files in `derivatives`

**Comment by lzehl** [comment]
@Peyman-N yes. Derivatives do not have (yet) a clear specification in BIDS and are therefore not reliably to capture in an automated conversion between BIDS datasets and openMINDS. Please ignore derivatives for now.
